core/market/decentralized/src/market.rs

```diff
@@ -114,7 +119,11 @@ impl MarketService {
     }
 
     pub async fn get_offers(&self, id: Option<Identity>) -> Result<Vec<Offer>, MarketError> {
-        Ok(self.matcher.store.get_offers(id).await?)
+        Ok(self
```

Comment:
I'm more and more concerned about why do we need those wrappers here in main market module. They should be covered within rest_api module. If you'll agree than please create issue based on that comment, as this should not be part of this PR

Reply:
I think, having it on market level isn't something bad. It is better api if you call `get_offers` instead dispatching `matcher.store...`.
But if you change it in future, I don't mind.

Reply:
ok, I've made an issue for it #530